samples/ChatbotExamplev2/README.md

```diff
@@ -0,0 +1,111 @@
+
+This sample shows how two Cognitive Services, LUIS and Personalizer, can be integrated into a coffee recommendation chat bot using ASP.NET Core 3.1.
+
+# Setting up the bot
+
+## Install Prereqs
+- [ASP.NET Core 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1)
+- [Microsoft Bot Framework Emulator](https://aka.ms/botframeworkemulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
+
+## Download the sample
+- Clone the samples repository
+```bash
+git clone https://github.com/Azure-Samples/cognitive-services-personalizer-samples.git
+```
+- Open [ChatbotSample.sln](./ChatbotExample.sln) in Visual Studio or VS Code
+
+## LUIS Migration
+- If you have a Language Understanding (LUIS) resource already created that you'd like to use, make sure you follow the LUIS migration steps [here](https://docs.microsoft.com/en-gb/azure/cognitive-services/luis/migration) if any of the scenarios listed apply.
+
+## Necessary Azure resources
+- Create both a Personalizer and a Language Understanding resource in the Azure portal. These will be used to power the reinforcement learning and NLP cababilities of the bot, respectively.
+- NOTE: when creating a LUIS resource in the portal, you can choose between creating a prediction resource, an authoring resource, or both. Only an authoring resource is needed for this sample. More on the two [here](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-concept-keys#azure-resources-for-luis)
+
+## Set up Personalizer
+- In [appsettings.json](./appsettings.json), modify `PersonalizerServiceEndpoint` to be your resource's endpoint and `PersonalizerAPIKey` to be either one of your resource's keys; this can be found in the Keys and Endpoint tab in the portal under `Endpoint` and `Key1`/`Key2`, respectively.
+
+## Set up LUIS
+- Navigate to [LUIS portal](https://www.luis.ai).
+- Click the `Sign in` button.
+- Click on `My Apps`.
+- Select the Subscription for your LUIS authoring resource and then the LUIS authoring resource.
+- Click on the `New app for conversation` button.
+- Click on `Import as JSON` and select [coffeebot.json](./CognitiveModels/coffeebot.json) from the `cognitive-services-personalizer-samples/samples/ChatbotExample/CognitiveModels` folder.
+
+- Click on the [`Train`](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-how-to-train) button on the top right of the page; this will train your app on the model inside of chatbot.json.
+
+- Once this is done, press the [`Publish`](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-how-to-publish-app) button and select "Production" (You can leave the Sentiment Analysis, Spell Check, and Speech Priming fields set to Off). NOTE: Once you publish your app on LUIS portal for the first time, it takes some time for the endpoint to become available, about 5 minutes of wait should be sufficient.
+
```

Comment:
It would be nice if this setup section about LUIS pointed out that the default key/endpoint is for trail purposes only, and has a limited amount and endpoint requests. If the user wants a more robust TPS, they need to create a LUIS resource and assign it to this app in the LUIS portal.

Reply:
added note